### PR TITLE
SVG importer: fix outlined rectangle and implement missing shapes

### PR DESCRIPTION
**Pull request by rodolforg**
Now few essential elements are missing: <text>, <image>, <filter> and <use>.

### CODE REVIEW

**Comment by rodolforg** [comment]
> 2 for FIXME comment

Just a reminder for another issue

File: synfig-core/src/modules/mod_svg/svg_parser.cpp [hunk 1]
```diff
@@ -1229,6 +1234,34 @@ Svg_parser::parser_path_circle(const xmlpp::Element* nodeElement, const Style& s
...
+	double y1 = 0;
+	double y2 = 0;
+
+	try {
+		x1 = std::stod(nodeElement->get_attribute_value("x1"));
+		x2 = std::stod(nodeElement->get_attribute_value("x2"));
+		y1 = std::stod(nodeElement->get_attribute_value("y1"));
+		y2 = std::stod(nodeElement->get_attribute_value("y2"));
+	} catch (...) {
+		synfig::error("SVG Parser: Invalid <line> attribute: x1,y1,x2,y2 should be real values or percentages!");
```
Comment (ice0) [inline synfig-core/src/modules/mod_svg/svg_parser.cpp:1301]: If the values can be percentages, will they be correctly interpreted?

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

File: synfig-core/src/modules/mod_svg/svg_parser.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-core/src/modules/mod_svg/svg_parser.cpp:1301]: Currently almost anywhere.
I'll provide a PR for fix at everywhere I can locate XD

**Pull request merged**